docs/userManual.md

```diff
@@ -657,9 +657,17 @@ containing two values, "true" and "false".
 
 Hillview can read data from [Cassandra distributed databases](https://cassandra.apache.org/).
 For this purpose a Hillview worker should be deployed on each Cassandra node.
-Moreover, Hillview must have read access to Cassandra's SSTables.
-Hillview assumes that no writes are in progress while reading the
-data from storage.
+Moreover, Hillview must have read access to Cassandra's SSTables. 
+To avoid reading inconsistent data, Hillview will read from a snapshot.
+More about Cassandra's snapshot can be found [here]
+(https://docs.datastax.com/en/dse/5.1/dse-admin/datastax_enterprise/tools/nodetool/toolsSnapShot.html).
```

Comment:
Could you give an example of what the user has to do to create a snapshot?
Also, how does the snapshot taken relate to the form they have to fill when loading the data?

Reply:
Updated with this snapshot command example.
![image](https://user-images.githubusercontent.com/7077197/92150271-c9ab6200-ede4-11ea-8ad9-1eef6f404f00.png)